code/src/components/TaskSummary.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export const TaskSummary = () => {
+  const list = useSelector(store => store.tasks.list);
+
+  const numDone = list.items.filter(item => item.complete).length;
+
+  return (
+    <section className="taskSummary">
+      <h2>
+        {numDone}/{list.items.length} TASKS DONE
```

Comment:
I would have been curious to see how many of the tasks in this category are done